Find a functionally equivalent code:android.hardware.SystemSensorManager.requestTriggerSensorImpl:COMMENT
Method Modifier: protected   hidden      
Comment:/**
 * @hide
 */

@@ -1,7 +1,7 @@
 {
     if (sensor == null)
         throw new IllegalArgumentException("sensor cannot be null");
-    if (Sensor.getReportingMode(sensor) != Sensor.REPORTING_MODE_ONE_SHOT)
+    if (sensor.getReportingMode() != Sensor.REPORTING_MODE_ONE_SHOT)
         return false;
     synchronized (mTriggerListeners) {
         TriggerEventQueue queue = mTriggerListeners.get(listener);

